src/vs/workbench/contrib/chat/browser/attachments/implicitContextAttachment.ts

```diff
@@ -79,6 +79,7 @@ export class ImplicitContextAttachmentWidget extends Disposable {
 		this._register(this.hoverService.setupManagedHover(getDefaultHoverDelegate('element'), hintElement, title));
 
 		const buttonMsg = this.attachment.enabled ? localize('disable', "Disable current file context") : localize('enable', "Enable current file context");
+		this.domNode.ariaLabel = buttonMsg;
```

Comment:
I think you can pass `ariaLabel: true` to the Button to use the title, maybe you want to do that? I'm not sure why that wouldn't be the default, if it's the best practice